dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.javax.jaxrs-2.0.feature

```diff
@@ -0,0 +1,16 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.javax.jaxrs-2.0
+visibility=private
+singleton=true
+IBM-App-ForceRestart: uninstall, \
+ install
+Subsystem-Name: Java RESTful Services API 2.0
+-features=com.ibm.websphere.appserver.javax.servlet-3.1, \
+ com.ibm.websphere.appserver.javax.annotation-1.2; apiJar=false, \
+ com.ibm.websphere.appserver.json-1.0, \
```

Comment:
this json-1.0 dependency doesn't need to be here.  It's still specified in the jaxrs-2.0 feature and isn't directly required by JAX-RS 2.0 API.  (Note this is the JSON4J feature, not JSON-P 1.0)